src/_data/contributors.json

```diff
@@ -0,0 +1,146 @@
+[
```

Comment:
Is there specific a criteria to meet to be considered for the contributors list?

Reply:
Sorry, I don't understand your question. Do you mean if there are mandatory criteria to fill in?

Reply:
I wasn't sure how the list was compiled since I'm not active in Discord but @mtom55 reached out and provided him my info, so all good now!